chart/kubeapps/values.yaml

```
@@ -22,6 +22,7 @@ ingress:
   # If certManager is set to true, annotation kubernetes.io/tls-acme: "true" will automatically be set
   annotations:
   #  kubernetes.io/ingress.class: nginx
+    ingress.kubernetes.io/proxy-read-timeout: "600"
```

I believe it is:

```suggestion
    nginx.ingress.kubernetes.io/proxy-read-timeout: "600"
```

as per https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/nginx-configuration/annotations.md#custom-timeouts

---

good catch, that was for a previous version